sdk/servicebus/service-bus/test/stress-test-track-2/scenarioSend.ts

```diff
@@ -18,7 +18,11 @@ interface ScenarioSimpleSendOptions {
 // Define connection string and related Service Bus entity names here
 const connectionString = process.env.SERVICEBUS_CONNECTION_STRING || "<connection string>";
 
-function sanitizeOptions(options: ScenarioSimpleSendOptions): Required<ScenarioSimpleSendOptions> {
+function sanitizeOptions(args: string[]): Required<ScenarioSimpleSendOptions> {
+  const options = parsedArgs<ScenarioSimpleSendOptions>(args, {
+    boolean: ["useScheduleApi"],
+    default: { useScheduleApi: false }
```

Comment:
Should line 31 be updated from `options.useScheduleApi || false` to `options.useScheduleApi` now?

Reply:
Doesn't matter since `<bool> || false = <bool>`.

Will update regardless, thanks!